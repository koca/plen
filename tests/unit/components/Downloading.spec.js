import { shallowMount } from "@vue/test-utils";
import Downloading from "@/components/Downloading.vue";

describe("Downloading.vue", () => {
  it("renders properly", () => {
    let wrapper;
    wrapper = shallowMount(Downloading);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
