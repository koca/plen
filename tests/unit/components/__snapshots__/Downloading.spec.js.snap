// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Downloading.vue renders properly 1`] = `
<div class="loading text-center bg-white fixed pin-t pin-l pin-r pin-b flex flex-col items-center justify-center">
  <div class="mb-8 text-black"><svg xmlns="http://www.w3.org/2000/svg" width="38" height="38" stroke="currentColor" viewBox="0 0 38 38">
      <g fill="none" fill-rule="evenodd" stroke-width="2" transform="translate(1 1)">
        <circle cx="18" cy="18" r="18" stroke-opacity=".5"></circle>
        <path d="M36 18c0-9.94-8.06-18-18-18">
          <animateTransform attributeName="transform" dur="1s" from="0 18 18" repeatCount="indefinite" to="360 18 18" type="rotate"></animateTransform>
        </path>
      </g>
    </svg></div>
  <p class="mb-8">Downloading the 30.000 sentences.<br>This may take couple seconds.</p>
  <p>Once downloaded,<br>this app also works <strong>offline</strong>.</p>
</div>
`;
